bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -161,7 +167,7 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/opendatahub/opendatahub-operator:dev-0.0.1
+                image: quay.io/vhire/opendatahub-operator:dev-0.0.1
```

Comment:
opendatahub vs vhire

Reply:
this has been fixed on the feature-rearchitecture branch, if rebase, should have it updated.